doc/tesseract.1.asc

```diff
@@ -277,7 +277,6 @@ following languages:
 *tat* (Tatar),
 *tel* (Telugu),
 *tgk* (Tajik),
-*tgl* (Tagalog),
```

Comment:
`tgl` is available for `tessdata` (so we should not remove it here), but is missing for `tessdata_fast` and `tessdata_best` which is strange. We could copy the LSTM part to `tessdata_fast` if that helps.

Reply:
> Please see https://tesseract-ocr.github.io/tessdoc/Data-Files-in-different-versions.html

Would you like me to also submit an update for that page?

In general it is not clear to me what the authoritative source on all the "officially" supported/included languages is. I used to think the man page, but now I assume the tessdata repos themselves.

In any case it's confusing when the documentation points to language data that does not exist - my first inclination was search ubuntu packages, where I did find some old ones, but importing those was likely not a good idea, which is why I started digging some more. If it helps I can try to automatically match all the other languages in the man page (or the wiki link) to the ones installed by ubuntu packaging and what is available in tessdata.

Reply:
@Shreeshrii - not sure why my comment went to this thread (I clicke quote reply), but my above comment is in reply to your message.